hazelcast/src/main/java/com/hazelcast/config/NativeMemoryConfig.java

```diff
@@ -67,7 +68,7 @@ public class NativeMemoryConfig {
     public static final int INITIAL_MEMORY_SIZE = MIN_INITIAL_MEMORY_SIZE;
 
     private boolean enabled;
-    private MemorySize size = new MemorySize(INITIAL_MEMORY_SIZE, MemoryUnit.MEGABYTES);
+    private Capacity size = new Capacity(INITIAL_MEMORY_SIZE, MemoryUnit.MEGABYTES);
```

Comment:
I don't have a strong opinion about this, but wouldn't the following lead to a cleaner API without hacks?

- rename the field to capacity, introduce new getter/setter for that
- introduce the new field to the xml/yaml/spring configs (this is the annoying bit, but it's easier because the NativeMemoryConfig is not dynamic, so no ser/de in client protocol)
- deprecate getSize/setSize methods, keep their signature (hence binary compatibility), and internally change the capacity field
- `memory-size` in xsd can be renamed to capacity, leaving one deprecated config with `<xs:element name="size" type="capacity" minOccurs="0"/>`, reversing the symmetry (now we have 3 `capacity` fields with type `memory-size`)

wdyt?

Reply:
@TomaszGaweda I am fine with both approaches (bridge method or add new `get/setCapacity`).

Notice that if you opt for the former, we need to keep `NativeMemoryConfig setSize(MemorySize)` method to maintain binary compat.